src/nyc_trees/apps/users/models.py

```diff
@@ -223,44 +221,47 @@ def __init__(self, name, description, description_achieved, badge,
     )),
     (AchievementDefinition.TREES_100, AchievementDefinition(
         name='Lavender Linden',
-        description='Map 100 Trees',
-        description_achieved='Mapped 100 Trees',
+        description='Map 100 Trees - 2016',
+        description_achieved='Mapped 100 Trees - 2016',
         badge='img/badges/Lavender-Linden.png',
         achieved=lambda user: get_user_tree_count(user, REWARD_START) >= 100,
         active=True,
-        reward="Reward in development. Stay tuned!"
+        reward="1, 2, Tree Shirts 2016 edition"
     )),
     (AchievementDefinition.TREES_300, AchievementDefinition(
         name='Magenta Maple',
-        description='Map 300 Trees',
-        description_achieved='Mapped 300 Trees',
+        description='Map 300 Trees - 2016',
+        description_achieved='Mapped 300 Trees - 2016',
         badge='img/badges/Magenta-Maple.png',
         achieved=lambda user: get_user_tree_count(user, REWARD_START) >= 300,
         active=True,
-        reward="Reward in development. Stay tuned!"
+        reward="""TreesCount! 2015 branded coffee tumbler.
+            Keeps your coffee warm or cold!"""
     )),
     (AchievementDefinition.TREES_500, AchievementDefinition(
         name='Silver Sophora',
-        description='Map 500 Trees',
-        description_achieved='Mapped 500 Trees',
+        description='Map 500 Trees - 2016',
+        description_achieved='Mapped 500 Trees - 2016',
         badge='img/badges/Silver-sophora.png',
         achieved=lambda user: get_user_tree_count(user, REWARD_START) >= 500,
         active=True,
-        reward="Reward in development. Stay tuned!"
+        reward="""TreesCount! 2015 branded sports duffel bag – The first 50
+            volun<b>treer</b>s to reach 500 trees since 5/11/2016 will receive
```

Comment:
Yeah, the rewards are counted starting from `5/11` but Parks had their first official mapping event on `5/13`. I talked with @dboyer and we decided to change the date.